src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+                        insights from the ReneCycle team!" -- Mina<br/>
+                        <br/>
+                        <b>LinkedIn:</b><br/>
+                        <Link href="https://www.linkedin.com/company/renecycle">https://www.linkedin.com/company/renecycle </Link><br/>
+                        <b>Website:</b><br/>
+                        <Link href="https://www.renecycle.com/">https://www.renecycle.com/</Link><br/>
+                        <b>Instagram:</b><br/>
+                        <Link href="https://www.instagram.com/renecycle.inc/">https://www.instagram.com/renecycle.inc/</Link>
+                    </Text>
+                </ColumnText>
+            </Columns>       
+        </div>
+    )
+}
+
+const SecondPlace = () => {
+    return (
+        <div>
+            <Subtitle css={`margin-top: 2vw`}>Second Place - Folia</Subtitle>
+            <Logo src={Folia} alt="Folia" css={`width: 20vw`}/>
+            <Text>Folia is a B2B Philadelphia-based company dedicated to diverting all 
+                food waste from the landfill. Through an online platform, they ensure quality 
+                matches between producers of food waste and food upcycling companies.</Text>
+            
+            <Columns css={`text-align: center`}>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/rosanna-jiang-a04139201/">
+                        <Headshot src={Rosanna} alt="Rosanna Jiang"/>                   
+                        <Name>Rosanna Jiang</Name>
+                    </Link>
+                </figure>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/caroline-l-185005128/">
+                        <Headshot src={Caroline} alt="Caroline Li" />                    
+                        <Name>Caroline Li</Name>
+                    </Link>
+                </figure>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/zacharyhuangogata/">
+                        <Headshot src={Zachary} alt="Zachary Huang-Ogata"/>
+                        <Name>Zachary <br/> Huang-Ogata</Name>
+                    </Link>
+                </figure>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/stephanie-b-ba88b1169/">
+                        <Headshot src={Stephanie} alt="Stephanie Bi"/>
+                        <Name>Stephanie Bi</Name>
+                    </Link>
+                </figure>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/ryomah/">
+                        <Headshot src={Ryoma} alt="Ryoma Harris"/>
+                        <Name>Ryoma Harris</Name>
+                    </Link>
+                </figure>
+            </Columns>
+
+            <Columns>
+                <ColumnText css={`padding-right: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">What was competing in the PCV Prize like?</Subtitle></b>
```

Comment:
Yup, you should add a bold prop to Subtitle that applies a font-weight style IMO